builders/invoke-workflow.psm1

```diff
@@ -0,0 +1,52 @@
+function Invoke-Workflow {
+    param (
+        [string] $Version,
+        [bool] $PublishRelease
+    )
+    
+    $payload = @{
+        "ref" = "main"
+        "inputs" = @{
+            "VERSION" = "$Version"
+            "PUBLISH_RELEASES" = "$PublishRelease"
+        }
+    } | ConvertTo-Json
+    $headers = @{
+        Authorization="Bearer $env:PERSONAL_TOKEN"
+    }
+    $uri = "$env:GITHUB_API_URL/repos/$env:GITHUB_REPOSITORY/actions/workflows/python-builder.yml/dispatches"
```

Comment:
I am mean to replace all occurrences(28,30,41 - lines) `"$env:GITHUB_API_URL/repos/$env:GITHUB_REPOSITORY/actions` in the script to the variable.

Reply:
I see. Makes sense.